lib/models.dart

```diff
@@ -37,3 +37,11 @@ class ExpenseTag {
   const ExpenseTag(
       {required this.tag, required this.expense, this.isSaved = true});
 }
+
+class ContactModel {
```

Comment:
Lets just name this Contact to keep the name parity with all other model names

Reply:
That library also have same name contact so when we use same name on contact screen it will not accept give error because of same name preset in both places which contact your are import that is not clear so